### PR TITLE
[docs] Updating ashell docs

### PR DESCRIPTION
**Pull request by brianjjones**
Signed-off-by: Brian Jones <brian.j.jones@intel.com>

### CODE REVIEW

File: docs/ashell.md [hunk 1]
```diff
@@ -77,19 +77,25 @@ acm> help
...
       ls [FILE] List directory contents or file stat
-     cat [FILE] Print the file contents on the stdout
+     cat [FILE] Print the file contents of a file
       du [FILE] Estimate file space usage
       rm [FILE] Remove file or directory
       mv [SOURCE] [DEST] Move a file to destination
+    test Runs your current test
+   error Prints an error using JerryScript
+    ping Prints '[PONG]' to check that we are alive
+      at OK used by the driver when initializing
```
Comment (grgustaf) [inline docs/ashell.md]: @sergioamr, old friend... can you tell us is this 'at' command there for some ATDT modem-related compatibility purpose, or just something we can drop? :)

File: docs/ashell.md [hunk 2]
```diff
@@ -77,19 +77,25 @@ acm> help
 
 Commands list:
     help This help
-    eval Evaluate JavaScript in real-time
+    eval Evaluate JavaScript in realtime
```
Comment (jimmy-huang) [inline docs/ashell.md]: real time?

File: src/ashell/shell-state.c [hunk 3]
```diff
@@ -601,8 +601,8 @@ static const struct ashell_cmd commands[] =
     ASHELL_COMMAND("rm",    "[FILE] Remove file or directory"                ,ashell_remove_file),
     ASHELL_COMMAND("mv",    "[SOURCE] [DEST] Move a file to destination"     ,ashell_rename),
 
-    ASHELL_COMMAND("rmdir", "[TODO]"                                         ,ashell_remove_dir),
-    ASHELL_COMMAND("mkdir", "[TODO]"                                         ,ashell_make_dir),
+//    ASHELL_COMMAND("rmdir", "[TODO]"                                         ,ashell_remove_dir),
```
Comment (jimmy-huang) [inline src/ashell/shell-state.c]: why not remove these if they are commented out?

Comment (brianjjones) [inline src/ashell/shell-state.c]: Because they will be used later

Comment (jimmy-huang) [inline src/ashell/shell-state.c]: then put a comment in there noting that they will be enabled later, so other developers don't have to guess

File: src/ashell/comms-shell.c [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016, Intel Corporation.
+// Copyright (c) 2016-2017, Intel Corporation.
```
Comment (grgustaf) [inline src/ashell/comms-shell.c]: @brianjjones, not comms-shell.c, which you didn't touch, but shell-state.c :)

File: src/ashell/shell-state.c (hunk 3 above)
Comment (brianjjones) [inline src/ashell/shell-state.c]: The TODO is there for that purpose, and the implementation calls printf("rmdir: Not implemented \n");

**Comment by grgustaf** [comment]
+1

File: docs/ashell.md (hunk 1 above)
Comment (sergioamr) [inline docs/ashell.md]: Hi! It was to answer back to the Linux driver when it tries to initialize the device.
The driver says AT and i answer back OK.

**Pull request merged**